.decent_ci.yaml

```diff
@@ -2,7 +2,7 @@ results_repository : Myoldmopar/EnergyPlusBuildResults
 results_path : _posts
 results_base_url : https://myoldmopar.github.io/EnergyPlusBuildResults
 regression_repository : NREL/EnergyPlusRegressionTool
-regression_branch : BumpToBoto3 # this is the branch of NREL/EnergyPlusRegressionTool to use (usually main)
+regression_branch : main # this is the branch of NREL/EnergyPlusRegressionTool to use (usually main)
```

Comment:
This should have been `main` a while back, as the `BumpToBoto3` branch was merged into main already.